### ISSUE TITLE
RA Normal and Turtle AIs try to build sub pen/ship yard when no water nearby

### ISSUE THREAD

**Issue by reaperrr** [OpenRA/OpenRA#8569]
The AI doesn't check for nearby water before starting to build them, leading to an infinite "try to place it - fail - try again after 5 seconds" loop.

As AI has some trouble handling water units properly anyway, the easiest "fix" for the time being would probably be to disable water production buildings (and units) for Normal and Turtle AI (Rush AI already doesn't use them). This would probably improve overall performance and efficiency of these two AIs until this is fixed properly.

**Comment by obrakmann** [comment]
Duplicate of #8116
